test/root/RootChain.test.js

```diff
@@ -62,24 +64,43 @@ contract('RootChain', async function (accounts) {
     }
   })
 
+  it('try to submitHeaderBlock with negative vote(`00` prefix) and fail', async function () {
+    const validators = [1, 2, 3, 4]
+    let tree = await rewradsTree(validators, accountState)
+    const { data, sigs } = buildSubmitHeaderBlockPaylod(
+      accounts[0],
+      0,
+      255,
+      '' /* root */,
+      wallets,
+      { allValidators: true, rewardsRootHash: tree.getRoot(), getSigs: true, totalStake: totalStake, sigPrefix: '0x00' }
+    )
+    try {
+      await rootChain.submitHeaderBlock(data, sigs)
```

Comment:
Please add `assert.fail('Expected the tx to revert')`, otherwise the test will pass even if the tx doesn't revert.